hedera-mirror-common/src/main/java/com/hedera/mirror/common/domain/transaction/UtilRandomGenerate.java

```diff
@@ -0,0 +1,61 @@
+package com.hedera.mirror.common.domain.transaction;
+
+/*-
+ * ‌
+ * Hedera Mirror Node
+ * ​
+ * Copyright (C) 2019 - 2022 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.domain.Persistable;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@AllArgsConstructor(access = AccessLevel.PRIVATE) // For Builder
+@Builder
+@Data
+@Entity
+@NoArgsConstructor
+public class UtilRandomGenerate implements Persistable<Long> {
+
+    @Id
+    private Long consensusTimestamp;
+
+    private byte[] pseudorandomBytes;
+
+    private int pseudorandomNumber;
```

Comment:
This can be nullable.

```suggestion
    private Integer pseudorandomNumber;
```

Reply:
I see, I have fixed it.